mlflow/shap/__init__.py

```diff
@@ -350,6 +350,7 @@ def log_explainer(
     """
 
     Model.log(
+        artifact_path=None,
```

Comment:
Can we pass `artifact_path`?

```suggestion
        artifact_path=artifact_path,
```

Reply:
I think I forgot to restore that. We first replaced `artifact_path` with `name` but restored `artifact_name` to avoid a breaking change.